test/html/test_html.py

```diff
@@ -699,3 +712,132 @@ def test_html_ol_ul_line_height(tmp_path):
     </ul>"""
     )
     assert_pdf_equal(pdf, HERE / "html_ol_ul_line_height.pdf", tmp_path)
+
+
+def test_html_long_list_entries(tmp_path):
+    pdf = FPDF()
+    pdf.add_page()
+    html = f"<ul><li>{'A ' * 200}</li></ul>"
+    pdf.write_html(html)
+    assert_pdf_equal(pdf, HERE / "html_long_list_entries.pdf", tmp_path)
+
+
+def test_html_long_ol_bullets(tmp_path):
+    pdf = FPDF()
+    pdf.add_page()
+    html_arabic_indian = f"""
+            <ol start="{10**100}">
+              <li>Item 1</li>
+              <li>Item 2</li>
+              <li>Item 3</li>
+            </ol>
+        """
+    html_roman = f"""
+            <ol start="{10**5}" type="i">
+              <li>Item 1</li>
+              <li>Item 2</li>
+              <li>Item 3</li>
+            </ol>
+        """
+    pdf.write_html(html_arabic_indian)
+    pdf.write_html(html_roman, type="i")
+    pdf.write_html(html_arabic_indian, tag_indents={"li": 50})
+    pdf.write_html(html_roman, tag_indents={"li": 100})
+    assert_pdf_equal(pdf, HERE / "html_long_ol_bullets.pdf", tmp_path)
+
+
+def test_html_measurement_units(tmp_path):
+    for unit in ["pt", "mm", "cm", "in"]:
+        pdf = FPDF(unit=unit)
+        pdf.add_page()
+        html = """
+                <ul>
+                    <li>Item 1</li>
+                    <li>Item 2</li>
+                    <li>Item 3</li>
+                </ul>
+                <ol>
+                    <li>Item 1</li>
+                    <li>Item 2</li>
+                    <li>Item 3</li>
+                </ol>
+                <blockquote>Blockquote text</blockquote>
+                <dt>Description title</dt>
+                <dd>Description details</dd>
+            """
+        pdf.write_html(html)
+        assert_pdf_equal(pdf, HERE / "html_measurement_units.pdf", tmp_path)
+
+
+def test_bulleted_paragraphs():
+    pdf = FPDF()
+    pdf.add_page()
+    text_columns = pdf.text_columns(skip_leading_spaces=True)
+    cases = [
+        {"indent": 1, "bullet_string": None},
+        {
+            "indent": 10,
+            "bullet_string": "",
+            "rel_x_displacement": 2,
+            "rel_y_displacement": 0,
```

Comment:
I think you eliminated "rel_y_displacement" from the code, so the name shouldn't appear in a test anymore.
"rel_x_displacement" should be named "bullet_r_margin" here as well to avoid confusion.
I also don't see the displacement/margin value actually being used in the test.